Set up 2D Camera Movement

Eagullfly opened the pull request:
Just put the player in Follow in the CinemachineVirtualCamera component and the Background in Bounding Shape 2D in Cinemachine Confiner. Let me know if you have any problems or suggestions.

Commit:
Set up 2D Camera Movement

Just put the player in Follow in the CinemachineVirtualCamera component and the Background in Bounding Shape 2D in Cinemachine Confiner. Let me know if you have any problems or suggestions.

BUNN1E5 commented on Assets/Scripts/BaseCode/Entity.cs:
```diff
@@ -10,7 +10,7 @@ public class Entity : MonoBehaviour
     public float health;
 
-    [EasyButtons.Button]
+    //[EasyButtons.Button]
```
Hey if possible you shouldn't change other people's code, this can cause a lot of problems if you change something and someone else made the same change.

Eagullfly replied:
Sorry, it was the only way I could implement cinemachine on my end before I updated Unity.

BUNN1E5 commented on Assets/Scripts/CameraShake.cs:
```diff
@@ -56,7 +56,7 @@ public class CameraShake : MonoBehaviour{
 
     // Other varibles like frequency could be add to this StartShake function later. 
-    [Button]
+    //[Button]
```
Was there a problem with Easy Buttons? I see your messages in help, but I wasn't in a situation to help you at that time. I can work with you to see what the problem is.

Eagullfly replied:
I couldn't find it in package manager, but ever since I upgraded to 2020.1.13f I can't access package manager. It shows me this.
![Package_Manager_Problem](https://user-images.githubusercontent.com/61594356/111020351-0c448d00-837a-11eb-863c-856a3c2e8ed3.PNG)

BUNN1E5 commented on ProjectSettings/ProjectVersion.txt:
```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2020.1.13f1
-m_EditorVersionWithRevision: 2020.1.13f1 (5e24f28bfbc0)
+m_EditorVersion: 2019.4.20f1
```
Hey you seem to be using the wrong version on Unity3d, try download version 2020.1.13f

Eagullfly replied:
Is it okay to use 2020.3.0f1?

BUNN1E5 commented on Packages/manifest.json:
```diff
@@ -1,21 +1,13 @@
-    "com.unity.test-framework": "1.1.18",
-    "com.unity.textmeshpro": "3.0.1",
-    "com.unity.timeline": "1.3.6",
+    "com.unity.test-framework": "1.1.22",
```
Hey if possible don't downgrade packages

Eagullfly replied:
I don't even know how I did that.

BUNN1E5 reviewed (changes requested):
I'm sorry, this code seems like it'll cause a lot of problems with the project. Me or @Kainkun can try to help you with any problems when we are available. But first please update your Unity Editor to version 2020.1.13f. 

Thank you.

There is the unfortunate possibility that you might need to start before the last commit and reimport cinemachine and setting that up.

Commit:
Updated Unity and uncommented code

I'm sorry for commenting other people's code. It was the only Unity would allow me to use cinemachine. For some reason I get an error with "using EasyButtons;" and I don't know why. Is there a package I need to implement? Anyway here it is again. Same instructions as the last commit.
